### DISCUSSION TITLE
Can't access any of my domain (404 : you are logged in as)

### DISCUSSION THREAD

**Discussion by zonetecde** [vercel/community#5627] [Help]
### Summary

Hello,

Today, I bought a new domain name and added it to Vercel. It worked well, but for the past hour, I haven't been able to access my domain settings anymore:
![image](https://github.com/vercel/community/assets/56195432/9ed9ef57-252b-493c-9269-fb36a876f801)

I am logged in, and this is the account from which I added the domain:
![image](https://github.com/vercel/community/assets/56195432/7fc54ad4-a78b-45a8-b273-f2a5051b1dd1)

This is weird; even if I switch browsers, try on my phone, or anything else, it won't let me go to the settings page.

Does anyone know how to solve this issue? Thank you.

P.S.: Of course, I did try "Sign in as a different user," but it still redirects me to this 404 page.
P.S.2: None of my domain settings work anymore, not only this one

### Example

https://vercel.com/dashboard/domains/

### Steps to Reproduce

1 - Add a domain
2 - Wait a few magic hours
3 - You can't access your domains anymore

**Comment by amyegan** [comment]
Thanks for reporting this. We're working to fix the broken link now. There is a suggested workaround in a related discussion post - https://github.com/vercel/community/discussions/5625#discussioncomment-8261722. Please let me know if that doesn't work for you 🙏
